### DISCUSSION TITLE
Accessing Inventory of a Shulker Box Inside of a Peripheral

### DISCUSSION THREAD

**Discussion by CubicLog** [cc-tweaked/CC-Tweaked#1492] [Q&A] [answered]
I have a shulker box bulk storage system. Each inventory contains shulker boxes filled with items. These inventories are hooked up as peripherals to a computer. Accessing each inventory peripheral, is there a way to tell what items are inside the shulker boxes? I need to be able to identify which inventory peripheral holds which type of items (inside the shulker boxes).
![image](https://github.com/cc-tweaked/CC-Tweaked/assets/70000984/530a886a-e085-47a4-a921-83ff3dec19eb)
(dispenser is the peripheral and I need to be able to identify what items are inside the shulker box)

**Comment by SirEdvin** [comment, marked as answer]
I guess, there is no way, you probably need some addon, that allow you to do so.

This is not pretty cool solution, but you can try use for this task [Item Pedestal](https://docs.siredvin.site/UnlimitedPeripheralWorks/peripherals/item_pedestal/) from UPW, but this requires you to move item, that you interested in to pedestal.

Or you can use Integrated Dynamics integration from UPW and just copy NBT information of dispenser to your computer.

**Comment by Lupus590** [comment]
Does each shulker box have a unique NBT hash and work as an inventory peripheral when placed?

If yes then you could place the box, look inside and then pick it up again. You then just need to match the name of the item inside with the NBT hash of the shulker box that those items are in.
